docs/insomnia/chaining-requests.md

```diff
@@ -5,57 +5,40 @@ category: "Requests and Responses"
 category-url: requests-and-responses
 ---
 
-Insomnia provides the ability to extract values from the responses of other requests. This is commonly referred to as chaining because the dependency upon another request forms a link.
+Insomnia allows chaining requests, or the ability to extract values from the responses of other requests. Values are passed using [Template Tags](/insomnia/template-tags) and [Environment Variables](/insomnia/environment-variables).
 
-Request chaining may be difficult to understand at first, especially if the concepts of [Template Tags](/insomnia/template-tags) or [Environment Variables](/insomnia/environment-variables) are unfamiliar. I suggest first taking a look at those if you are having trouble getting started.
+The following are common use cases for chaining requests:
 
-First, let's take a look at some examples that chaining may be useful for.
+* use a token from a login response as an Authorization header
+* use the ID from POST `/items` in PUT `/items/{id}`
+* use an entire body from one response in another request
 
-## Common Uses
+## Select Response Tag
 
-To help explain request chaining, here are some common use cases:
+In an empty URL bar or following a URL symbol such as `/` or `=`, either press CTRL+Space to open the autocomplete dropdown or start typing "response".
 
-* use token from login response as an Authorization header
-* use id from POST `/item in PUT /items/<id>`
-* use entire body from one response to upload to another
+There are three response options:
 
-There are many other ways to use chaining, but those are the most commonly used examples.
+* **Response - Body Attribute**
+  * Works on JSON and XML responses, pulling attributes from a response body.
 
-## Chaining Basics
+    Use either JSONPath or XPath queries (depending on the Content-Type) to extract specific attributes from a response body. For example, a login request may respond with a JSON object that looks like `{"auth_token": "secret-token"}`. This allows you to use a JSONPath filter of `$.auth_token` to reference the token itself in a subsequent request.
+* **Response - Raw Body**
+  * References the entire body contents of a request. This can be useful if you download content in one request that needs to be uploaded in another.
+* **Response - Header**
+  * Pulls specific header values out of a response.
 
-To get started, select one of three Response tags from the autocomplete dropdown. You can invoke the dropdown be pressing CTRL+Space or by typing starting characters of the tag (respo...). The response tag can be used anywhere that environment variables are supported (URL, headers, request body, environments, etc).
+The Response tag can be used anywhere [Environment Variables](https://docs.insomnia.rest/insomnia/environment-variables) are supported.
```

Comment:
We have a line in the env vars page that says "Environment variables can be referenced in (almost) any text input within of the Insomnia application.". Perhaps we could paraphrase that here instead of having users ask "well, where can env vars be used"?

(Relates to my other comment)